hs_file_types/models/base.py

```diff
@@ -25,10 +25,7 @@
     set_dirty_bag_flag, add_file_to_resource, resource_modified, get_file_from_irods
 from hs_core.models import ResourceFile, AbstractMetaDataElement, Coverage
 from hs_core.signals import post_remove_file_aggregation
-
-RESMAP_FILE_ENDSWITH = "_resmap.xml"
-METADATA_FILE_ENDSWITH = "_meta.xml"
-SCHEMA_JSON_FILE_ENDSWITH = "_schema.json"
```

Comment:
might want to update the `is_map_file()` here:
https://github.com/hydroshare/hydroshare/blob/5473-aggregation-meta-file-download-fix/hs_file_types/utils.py#L314-L315

Reply:
I assume use the new enum types for aggregation meta file extensions to replace the '_resmap.xml'? Unless we also replace the 'resourcemap.xml' using another enum type, I think it is better to not change.

Reply:
sounds good!